### DISCUSSION TITLE
Integration with MAUI .NET Multi-platform App UI

### DISCUSSION THREAD

**Discussion by malix0** [gircore/gir.core#366] [Q&A]
Is there a plan to integrate this with MAUI https://github.com/dotnet/maui ?

**Comment by badcel** [comment]
Yes there were already some discussions in different Maui related repos. But currently we need to stabilise our code base and API which will take some time.

The latest statement can be found [here](https://github.com/jsuarezruiz/maui-linux/issues/8#issuecomment-831419497) .

If it should happen some time the integration would probably happen in the Maui project. But I think we could support this process. Currently it is to early for any work in this direction.

**Comment by malix0** [comment]
Thanks for your answer and excuse me if I open an issue and not come here. I'm really interested to MAUI under Linux, because in my opinion is the only way we can get Visul Studio (not VSCode) under Linux.

I just posted a message on Vs comunity forum, under "Visual Studio for Linux" thread, asking someone to help this project and MAUI for Linux https://developercommunity.visualstudio.com/t/Visual-Studio-for-Linux/360479#T-N1466604
